commit cae3d71212ff1885d9317c22411fe97ccd99e287
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Sep 3 14:33:35 2017 +0200

    hurd: Fix rtld's strtoul_internal use through hurdlookup
    
            * sysdeps/mach/hurd/dl-sysdep.c (__strtoul_internal): New
            function.

diff --git a/sysdeps/mach/hurd/dl-sysdep.c b/sysdeps/mach/hurd/dl-sysdep.c
index fd2f3d52cb..517e4d62cc 100644
--- a/sysdeps/mach/hurd/dl-sysdep.c
+++ b/sysdeps/mach/hurd/dl-sysdep.c
@@ -602,6 +602,15 @@ __sbrk (intptr_t increment)
   return (void *) addr;
 }
 
+unsigned long int
+weak_function
+__strtoul_internal (const char *nptr, char **endptr, int base, int group)
+{
+  assert (base == 0 || base == 10);
+  assert (group == 0);
+  return _dl_strtoul (nptr, endptr);
+}
+
 void weak_function attribute_hidden
 _exit (int status)
 {
